tmail-backend/integration-tests/jmap/distributed-jmap-openpaas-integration-tests/pom.xml

```diff
@@ -0,0 +1,182 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
As I have said I would have had prefer us NOT to create a separate module or a separate configuration file.
I believe this choice is accidental.
Multiplication of maven modules is detrimental and needs to be fight.
Is there some new reasons compared to our thursday meeting about adding this module?

Reply:
My bad, I misunderstood the comment in https://github.com/linagora/tmail-backend/pull/1554#discussion_r1971448563